lib/gigya_flutter_plugin.dart

```diff
@@ -219,4 +219,67 @@ class GigyaSdk {
   }) {
     return GigyaFlutterPluginPlatform.instance.sso(parameters: parameters);
   }
+
+  /// Check isLocked by biometrics.
+  Future<bool> isLocked() {
+    return GigyaFlutterPluginPlatform.instance.isLocked();
+  }
+
+  /// Check isOptIn to biometrics.
+  Future<bool> isOptIn() {
+    return GigyaFlutterPluginPlatform.instance.isOptIn();
+  }
+
+  /// Opt into biometrics.
+  ///
+  /// @parameters The parameters passed in here is for the GigyaInfoPrompt.
```

Comment:
Dartdoc uses `[parameter]` as documentation for parameters. The parameters in the map are not required, but optional. Perhaps also add links to the SAP SDK so that users can look into parameter docs there?
```suggestion
  /// Opt into biometrics.
  ///
  /// The [parameters] are passed to the Biometrics SDK.
```

This remark also applies to `optOut` and `unlockSession` below.